PMM-12545-dbaas-removal-ui (FB)

YashSartanpara1 opened the pull request:
Custom branches: 
grafana

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

dependabot[bot] committed:
chore(deps): bump gitpython from 3.1.36 to 3.1.37 (#3396)

Bumps [gitpython](https://github.com/gitpython-developers/GitPython) from 3.1.36 to 3.1.37.
- [Release notes](https://github.com/gitpython-developers/GitPython/releases)
- [Changelog](https://github.com/gitpython-developers/GitPython/blob/main/CHANGES)
- [Commits](https://github.com/gitpython-developers/GitPython/compare/3.1.36...3.1.37)

---
updated-dependencies:
- dependency-name: gitpython
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
PMM-7 fixed "removed" call (#3402)

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
Bump VERSION (#3406)

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

BupycHuk committed:
PMM-7 update percona-toolkit and mongodb_exporter (#3439)

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

dependabot[bot] committed:
chore(deps): bump actions/github-script from 6 to 7 (#3452)

Bumps [actions/github-script](https://github.com/actions/github-script) from 6 to 7.
- [Release notes](https://github.com/actions/github-script/releases)
- [Commits](https://github.com/actions/github-script/compare/v6...v7)

---
updated-dependencies:
- dependency-name: actions/github-script
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

yurkovychv committed:
PMM-7 update GH status action repo (#3457)

Co-authored-by: Vadim Yarosh <vadym.yarosh@percona.com>

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
PMM-11609 removed bats (#3459)

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
PMM-12622 added pmm-server only tests and quiet mode (#3458)

* PMM-12622 added pmm-server only tests and quiet mode

* PMM-12622 updated health check

* PMM-12622 removed browser install

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

YashSartanpara1 committed:
Create feature build: PMM-12545-dbaas-removal-ui

JNKPercona commented:
server docker - perconalab/pmm-server-fb:PR-3474-5c2176e
client docker - perconalab/pmm-client-fb:PR-3474-5c2176e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3474-5c2176e.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3474-5c2176e&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3474-5c2176e.tar.gz

JNKPercona commented:
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14621/

YashSartanpara1 committed:
Trigger FB

puneet0191 committed:
PMM-7 Update VERSION 2.41.1 (#3477)

Commit:
chore: rewind submodules for dev-latest

JNKPercona commented:
server docker - perconalab/pmm-server-fb:PR-3474-93eecce
client docker - perconalab/pmm-client-fb:PR-3474-93eecce
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3474-93eecce.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3474-93eecce&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3474-93eecce.tar.gz

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

Commit:
chore: rewind submodules for dev-latest

dependabot[bot] committed:
chore(deps): bump actions/upload-artifact from 3 to 4 (#3487)

Bumps [actions/upload-artifact](https://github.com/actions/upload-artifact) from 3 to 4.
- [Release notes](https://github.com/actions/upload-artifact/releases)
- [Commits](https://github.com/actions/upload-artifact/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/upload-artifact
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump gitpython from 3.1.37 to 3.1.40 (#3422)

Bumps [gitpython](https://github.com/gitpython-developers/GitPython) from 3.1.37 to 3.1.40.
- [Release notes](https://github.com/gitpython-developers/GitPython/releases)
- [Changelog](https://github.com/gitpython-developers/GitPython/blob/main/CHANGES)
- [Commits](https://github.com/gitpython-developers/GitPython/compare/3.1.37...3.1.40)

---
updated-dependencies:
- dependency-name: gitpython
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

YashSartanpara1 committed:
Merge branch 'PMM-2.0' into PMM-12545-dbaas-removal-ui

JNKPercona commented:
server docker - perconalab/pmm-server-fb:PR-3474-be24a08
client docker - perconalab/pmm-client-fb:PR-3474-be24a08
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3474-be24a08.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3474-be24a08&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3474-be24a08.tar.gz

JNKPercona commented:
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/14681/

JNKPercona commented:
server docker - perconalab/pmm-server-fb:PR-3474-be24a08
client docker - perconalab/pmm-client-fb:PR-3474-be24a08
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3474-be24a08.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3474-be24a08&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3474-be24a08.tar.gz
